BillTrack.Worker/Services/InvoicePdfUploader.cs

```diff
@@ -0,0 +1,30 @@
+using Amazon.S3;
+using Amazon.S3.Model;
+using BillTrack.Core.Interfaces.Repositories;
+using BillTrack.Core.Interfaces.Services;
+using BillTrack.Domain.Entities;
+
+namespace BillTrack.Worker.Services;
+
+public class InvoicePdfUploader : IFileUploader
```

Comment:
yes, this service is for "uploading". An invoice could be any file type. The type should be a specific method (function takes and returns exact types).
This name is closer to "InvoiceUploader". Its methods will handle "where upload"/"what service should be used for an upload", file types for each upload.